### PR TITLE
docs: update references to SDK/Crawlee + more

### PR DESCRIPTION
**Pull request by AndreyBykov**
- Updated links, mentioned Crawlee where applicable, etc
- Tried to update some unrelated things which I noticed on the go (e.g. notice about migration to new billing, etc)
- Added some TODOs, e.g. not to forget to update example code in different repo and so on.
- All tutorials -> actor-scraper should be updated. Currently they are [here](https://github.com/apify/actor-scraper/tree/master/docs/build).

I did not go through api docs (except replacing existing token to some non-existing one, which was in some places there already).

I did not add any new content/significant updates.

### CODE REVIEW

**Comment by m-murasovs** [comment]
@AndreyBykov the token is definitely an accident - could you please change it?

**Comment by AndreyBykov** [comment]
@m-murasovs it seems it was there for quite a while. I would remove it from examples with something like `<YOUR_APIFY_TOKEN>`, but not sure about resetting it for the acc - I am not sure whether it used somewhere else explicitly or not...

File: content/docs/access_rights/list_of_permissions.md
Comment (m-murasovs) [inline content/docs/access_rights/list_of_permissions.md:109]:
```suggestion
## [](#custom-solutions) Custom Solutions
```

File: content/docs/access_rights/organization_account.md
Comment (m-murasovs) [inline content/docs/access_rights/organization_account.md:52]: No need for a comma here
```suggestion
And that's it! Your personal account becomes the organization and you will be logged out automatically. You can now log into your new personal account with the same credentials as you are currently logged in with. This applies to both **password** and **OAuth** methods.
```

File: content/docs/access_rights/organization_account/setup.md
Comment (m-murasovs) [inline content/docs/access_rights/organization_account/setup.md:33]: No need to remove this, it's clearer with the 'of'
```suggestion
You can configure individual permissions for each resource type such as actors, actor tasks or storage. Bear in mind that if a user has the **read** permission for [storages]({{@link storage.md}}), you cannot prevent them from accessing a particular storage (e.g. a certain [dataset]({{@link storage.md}})) - they will have access to all of the organization's storages.
```

File: content/docs/actors/development/testing_and_maintenance.md
Comment (m-murasovs) [inline content/docs/actors/development/testing_and_maintenance.md:30]:
```suggestion
We also advise to test your actor's default run – one that uses the pre-filled inputs. It is often the first task your users run and they may be put off if it doesn't work.
```

File: content/docs/actors/paid_actors.md
Comment (m-murasovs) [inline content/docs/actors/paid_actors.md:46]:
```suggestion
If you have an [Apify paid plan](https://apify.com/pricing), you will be automatically charged the monthly rental fee at the end of your free trial and you will be able to run the actor for another month. If you aren't subscribed to an Apify plan, you will need to subscribe to one of [Apify's paid plans](https://apify.com/pricing) in order to continue using the actor after the trial has ended.
```

File: content/docs/actors/paid_actors.md
Comment (m-murasovs) [inline content/docs/actors/paid_actors.md:72]:
```suggestion
Paid actors are still in **beta** stage and you will likely be one of this feature's very first users. So, if you have any feedback which is not actor-specific, but relates more to the experience of renting an actor or similar topics, please let us know at [community@apify.com](mailto:community@apify.com). Thank you!
```

File: content/docs/actors/publishing.md
Comment (m-murasovs) [inline content/docs/actors/publishing.md:51]:
```suggestion
**IMPORTANT:** Note that if you make an actor publicly available and it's used by other people, their usage is not charged towards your account. The user running the actor is always the one who pays for the computational resources consumed by an actor's execution.
```

File: content/docs/actors/publishing/seo_and_promotion.md
Comment (m-murasovs) [inline content/docs/actors/publishing/seo_and_promotion.md:13]:
```suggestion
Work on your actors does not end when you publish it. Now, it's time to promote your creation and get it to its intended audience. The Apify domain will boost your position with search engines but it won't do all the work. Focus on crafting content according to topics and phrases people search for to give your actor that extra boost.
```

File: content/docs/actors/publishing/seo_and_promotion.md
Comment (m-murasovs) [inline content/docs/actors/publishing/seo_and_promotion.md:80]:
```suggestion
If you’re not sure about similar phrases, check out the Google suggest tool in the search and it will show you some hints and tips. Of course, the description should primarily be about what the actor does, but it’s always a good idea to enrich the text with popular keywords.
```

File: content/docs/monitoring/monitor_shared_datasets.md
Comment (m-murasovs) [inline content/docs/monitoring/monitor_shared_datasets.md:48]: We reserve the code marks just for code, so for text, I'd prefer to use the double asterisk. Unless this name is in code, in which case we have a dilemma :D 
```suggestion
> We recommend prefixing your monitoring task names with **monitoring-** so you could identify them easier.
```

File: content/docs/actors/development/source_code.md
Comment (m-murasovs) [inline content/docs/actors/development/source_code.md:127]:
```suggestion
The actor owner can specify custom environment variables that are set to the actor's process during the run. Sensitive environment variables such as passwords or API tokens can be protected by setting the **Secret** option. With this option enabled, the value of the environment variable is encrypted and it will not be visible in the app or APIs. In addition, the value is redacted from actor logs to avoid the accidental leakage of sensitive data.
```

File: content/docs/storage/request_queue.md
Comment (m-murasovs) [inline content/docs/storage/request_queue.md:62]:
```suggestion
The default request queue's ID is **default**. Each request in the queue is stored as a separate JSON file, where {ID} is a request ID.
```

File: content/docs/tutorials/scrape_paginated_sites.md
Comment (m-murasovs) [inline content/docs/tutorials/scrape_paginated_sites.md:70]:
```suggestion
We have the base algorithm but before we start coding, let's answer a few questions to get more insight.
```

File: content/docs/tutorials/integrations/run_actor_and_retrieve_data_via_api.md
Comment (m-murasovs) [inline content/docs/tutorials/integrations/run_actor_and_retrieve_data_via_api.md:55]:
```suggestion
We can also add settings for the actor (which will override the default settings) as additional query parameters. For example, if we wanted to change how much memory the actor's run should be allocated and which build to run, we could simply add the `memory` and `build` parameters separated by `&`.
```

**Review by m-murasovs — CHANGES_REQUESTED** [review]
Thank you for all the fixes and improvements - this is a HUGE help! ❤️  ❤️  ❤️  ❤️ 

I have made some suggestions for some minor stuff, like the comma before 'and' and 'but' where it's not needed. I'm going off of [this article](https://www.grammarly.com/blog/comma-before-and/). The rest looks great!

**Comment by AndreyBykov** [comment]
@m-murasovs I was mainly following WebStorm suggestions in these things :) Will merge your suggestions and then will finish with last commit (if you would be able to check few last docs).

File: content/docs/access_rights/organization_account.md
Comment (AndreyBykov) [reply content/docs/access_rights/organization_account.md:52]: @m-murasovs I don't wanna be annoying, but isn't it exactly the case frmo the article, lol?

> Here’s a tip: Remember, when you’re joining two independent clauses, you need both a comma and a conjunction. If you use a comma without a conjunction, you’ll end up with a [comma splice](https://www.grammarly.com/blog/comma-splice/).

> Let’s look at another example.
It’s cold outside, and I can’t find my coat.

File: content/docs/actors/development/testing_and_maintenance.md
Comment (AndreyBykov) [reply content/docs/actors/development/testing_and_maintenance.md:30]: same as above =D

File: content/docs/access_rights/organization_account.md
Comment (m-murasovs) [reply content/docs/access_rights/organization_account.md:52]: Yeah, fair point 👍

File: content/docs/actors/development/testing_and_maintenance.md
Comment (m-murasovs) [reply content/docs/actors/development/testing_and_maintenance.md:30]: Technically yeah, but somehow it doesn't feel right to me with a comma. I'll leave it up to you though :)

**Comment by m-murasovs** [comment]
> I would remove it from examples with something like <YOUR_APIFY_TOKEN>

@AndreyBykov yes please!

**Comment by AndreyBykov** [comment]
@m-murasovs so should be done. The same token was used in api docs, but there was also some other one, currently non-existing - so I replaced it everywhere in api docs with this non-existing one.

Last batch is the files without folders - directly in `content/docs`, feesback welcome :)

**Review by m-murasovs — APPROVED** [review]
Amazing, thank you! ❤️ 🍻 ❤️

**Review by mstephen19 — APPROVED** [review]

**Comment by mstephen19** [comment]
Great work @AndreyBykov :D

**Comment by AndreyBykov** [comment]
We agreed with @B4nan that I'll merge it, and if someone would have some comments - let's fix it in next PRs.

**Pull request merged**
